### PR TITLE
Migrating WT3 municipal wwtp model

### PR DESCRIPTION
**Pull request by luohezhiming**
## Summary/Motivation:
 Add WT3 municipal wwtp model

## Changes proposed in this PR:
- add municipal wwtp model originally developed by @MarkusHolly
- add costing model
- add corresponding tests

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/zero_order/tests/test_municipal_wwtp_zo.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            "database": m.db})
+
+        m.fs.unit.inlet.flow_mass_comp[0, "H2O"].fix(1000)
+        m.fs.unit.inlet.flow_mass_comp[0, "foo"].fix(1)
+
+        return m
+
+    @pytest.mark.unit
+    def test_build(self, model):
+        assert model.fs.unit.config.database == model.db
```
Comment (kurbansitterley) [inline watertap/unit_models/zero_order/tests/test_municipal_wwtp_zo.py]:
```suggestion
        assert model.fs.unit.config.database is model.db
```

**Review by kurbansitterley — DISMISSED** [review]
just changed the `==` to `is` in the test, LGTM

**Comment by luohezhiming** [comment]
> just changed the `==` to `is` in the test, LGTM

@kurbansitterley I don't think we need to change all `==` to `is`, if the object for comparison is just some value, it is fine to use `==`. If the object is some other instance like `model.db`, then we'd better use `is`.

**Comment by kurbansitterley** [comment]
> > just changed the `==` to `is` in the test, LGTM
> 
> @kurbansitterley I don't think we need to change all `==` to `is`, if the object for comparison is just some value, it is fine to use `==`. If the object is some other instance like `model.db`, then we'd better use `is`.

I agree, no need to change all `==`, just applying some guidance I got for this particular test. Sorry, wasn't suggesting that all `==` need to be changed to `is` only commenting on the suggested change I made.

**Comment by luohezhiming** [comment]
> @pytest.mark.unit
>     def test_build(self, model):
>         assert model.fs.unit.config.database == model.db

OK. The change is already commited and resolved, I missed this, sorry about that.

**Review by bknueven — DISMISSED** [review]

**Review by andrewlee94 — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]

**Review by bknueven — APPROVED** [review]

**Pull request merged**